src/openslide-vendor-dicom.c

```diff
@@ -101,15 +105,20 @@ static const struct allowed_types LEVEL_TYPES = {
 // the ImageTypes we allow for associated images
 static const char LABEL_TYPE[] = "LABEL";
 static const char OVERVIEW_TYPE[] = "OVERVIEW";
+static const char THUMBNAIL_TYPE[] = "THUMBNAIL";
 static const char *const LABEL_TYPES[] = {
   "ORIGINAL", "PRIMARY", LABEL_TYPE, "NONE", NULL
 };
 static const char *const OVERVIEW_TYPES[] = {
   "ORIGINAL", "PRIMARY", OVERVIEW_TYPE, "NONE", NULL
 };
+static const char *const THUMBNAIL_TYPES[] = {
+  "DERIVED", "PRIMARY", THUMBNAIL_TYPE, "RESAMPLED", NULL
```

Comment:
Is it possible that we'd see `ORIGINAL` thumbnails?

Reply:
I don't think so -- the thumbnail is generally the lowest pyr level, so it will have been resampled.

Reply:
Ah that's a good correction. Yes, `LOCALIZER` was bafflingly useless. I've added DERIVED too.